x/stake/commands/tx.go

```diff
@@ -30,10 +30,20 @@ func GetCmdDeclareCandidacy(cdc *wire.Codec) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			pk, err := GetPubKey(viper.GetString(FlagPubKey))
+
+			pkStr := viper.GetString(FlagPubKey)
+			if len(pkStr) == 0 {
+				return fmt.Errorf("must use --pubkey flag")
+			}
+			pkBytes, err := hex.DecodeString(pkStr)
 			if err != nil {
 				return err
 			}
+			pk, err := crypto.PubKeyFromBytes(pkBytes)
```

Comment:
Would we want the option to pass a hex-encoded pubkey for any reason?

Reply:
It is standard to pass in hex encoded bytes as an argument for flags in CLI... certainly doesn't make sense to pass in JSON in the CLI! .. a bit confused by your comment .. it needs to be hex right

Reply:
I mean hex-encoded, non Amino-encoded pubkeys (what was previously implemented)

Reply:
well - maybe in another context in the future - for now I don't want to introduce multiple flags to feed the pubkey in... I think it's not so bad though, if you only have non-amino hex (and it's an ed25519 key) you can just manually prepend the string "1624de6220" to the beginning of your hex if you're not using golang - I'll add that information to the flag comment